src/lp-form.js

```diff
@@ -77,6 +78,7 @@ function lpForm (options={}) {
         submitFilters,
         initialValuesFilters,
         constraints={},
+        validateOptions={},
```

Comment:
So this is the smallest of potatoes but could you rename to `validationOptions`? As it stands the key sounds a bit like a boolean flag.

Otherwise this is good to go!!